components/script/dom/xrview.rs

```diff
@@ -28,6 +31,9 @@ pub struct XRView {
     #[no_trace]
     view: View<ApiSpace>,
     transform: Dom<XRRigidTransform>,
+    viewport_modifiable: Cell<bool>,
+    requested_viewport_scale: Cell<f64>,
+    current_viewport_scale: Cell<f64>,
```

Comment:
If this member is never read or written after the constructor, can we remove it?